src/EditorFeatures/DiagnosticsTestUtilities/CodeActions/CSharpCodeRefactoringVerifier`1+Test.cs

```diff
@@ -62,9 +62,9 @@ public Test()
 
             /// <summary>
             /// Gets or sets the language version to use for the test. The default value is
-            /// <see cref="LanguageVersion.CSharp8"/>.
+            /// <see cref="LanguageVersion.CSharp9"/>.
             /// </summary>
-            public LanguageVersion LanguageVersion { get; set; } = LanguageVersion.CSharp8;
+            public LanguageVersion LanguageVersion { get; set; } = LanguageVersion.CSharp9;
```

Comment:
this will end up changing *all* tests that use this.  I'd be wary about making this change.  Instead, consider only setting this in your specific test (or all the tests in that area ("move static")).